src/client/utils/download/download.ts

```diff
@@ -46,10 +46,11 @@ export function download({ dataset, options }: DataSetWithTabOptions, fileFormat
 }
 
 export function datasetToFileString(dataset: Dataset, fileFormat: FileFormat, options?: TabulatorOptions): string {
+  const optionsCopy = options ? { ...options } : {};
```

Comment:
What is the reason that we need options copy? This is not obvious and should be commented.